From fa3af966032267e618b19bbf06a536ddb81ddbdf Mon Sep 17 00:00:00 2001
From: Peter Oh <peter.oh@bowerswilkins.com>
Date: Tue, 29 May 2018 14:39:13 -0700
Subject: [PATCH 09/18] mesh: set mesh center frequency

vht center frequency value is required to compose the correct channel info.

Signed-off-by: Peter Oh <peter.oh@bowerswilkins.com>
---
 wpa_supplicant/mesh.c | 1 +
 1 file changed, 1 insertion(+)

--- a/wpa_supplicant/mesh.c
+++ b/wpa_supplicant/mesh.c
@@ -457,6 +457,7 @@ int wpa_supplicant_join_mesh(struct wpa_
 
 	if (wpa_s->mesh_vht_enabled) {
 		ssid->vht = 1;
+		ssid->vht_center_freq1 = params->freq.center_freq1;
 		switch (params->freq.bandwidth) {
 		case 80:
 			if (params->freq.center_freq2) {
